server/series/api.py

```diff
@@ -265,6 +279,101 @@ def revoke_series_invitation(
     return 200, invitation
 
 
+@router.get(
+    "/accept-invitation",
```

Comment:
can we combine this whole api to existing accept api as mostly both are doing same work? we can accept either token or inivitation_id in body / query params and do the same stuff after that?

Reply:
Although discussed, putting it here for others to see.

Since one is validating via CSRF and other is validating via JWT we cannot merge both the functions.